weed/command/filer_sync.go

```diff
@@ -126,6 +146,26 @@ func runFilerSynchronize(cmd *Command, args []string) bool {
 	return true
 }
 
+func initOffsetFromTsMs(grpcDialOption grpc.DialOption, sourceFiler pb.ServerAddress, targetFiler pb.ServerAddress, fromTsMs int) error {
+	if fromTsMs <= 0 {
+		return nil
+	}
+	// convert to nanosecond
+	fromTsNs := int64(fromTsMs * 1000_000)
+	// read source filer signature
+	sourceFilerSignature, sourceErr := replication.ReadFilerSignature(grpcDialOption, sourceFiler)
```

Comment:
please try to avoid repeating the code.

Reply:
you can change the old logic and only read the filer signatures when the sync process starts.

Reply:
OK, there are too many commit. I'll change the edition and mention pr again. I'll turn this off first. Thanks.